extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -479,4 +483,19 @@ export class DataExtensionsEditorView extends AbstractWebview<
       message: "",
     });
   }
+
+  private async callApi(request: ModelRequest): Promise<ModelResponse | null> {
```

Comment:
Can we make this name a little less generic? The data extensions editor isn't only for auto-modeling, so it's not clear from the name which API this is calling.

Reply:
🤦‍♀️